tests/functional/d/deprecated/deprecated_module_enabled.rc

```diff
@@ -0,0 +1,3 @@
+[Messages Control]
+disable=all
+enable=deprecated-module
```

Comment:
This disable/enable should be done automatically when running with ``--minimal-messages-config``. Is there a reason we include this explicitly here? (Also applies to tests/functional/d/deprecated/deprecated_module_redundant.rc)

Reply:
Ah, it was because we don't run `--minimal-messages-config` automatically. But if we're on the verge of turning that on I can remove these.

Reply:
Yes, that would be great! I will prepare a PR to include it in the CI.